src/osd/PrimaryLogPG.cc

```diff
@@ -1844,15 +1844,9 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
   ceph::ref_t<Session> session;
   if (can_backoff) {
     session = static_cast<Session*>(m->get_connection()->get_priv().get());
-    if (!session.get()) {
-      dout(10) << __func__ << " no session" << dendl;
-      return;
-    }
-
-    if (session->check_backoff(cct, info.pgid, head, m)) {
```

Comment:
I think you by accident dropped the backoff section too?

Reply:
check_backoff used by do_request.
    if (msg_type == CEPH_MSG_OSD_OP) {
      if (session->check_backoff(cct, info.pgid,
                                 info.pgid.pgid.get_hobj_start(), m)) {
        return;
      }

Reply:
@majianpeng pg-wide backoff is different from per-object backoff.

Reply:
Yes, you are right.  I missed this . I'll modify.